lib/modules/dosomething/dosomething_signup/dosomething_signup.install

```diff
@@ -710,3 +715,19 @@ function dosomething_signup_update_7024() {
     db_add_field($tbl_name, $field_name, $schema[$tbl_name]['fields'][$field_name]);
   }
 }
+
+/**
+ * Adds transactional column to the dosomething_signup table.
+ */
+function dosomething_signup_update_7025() {
```

Comment:
This will create a `transactional` field on every signup, I don't think we need to store if the signup was transactional on each individual signup record?